js/src/dropdown.js

```diff
@@ -111,7 +111,7 @@ const Dropdown = (($) => {
       // https://www.quirksmode.org/blog/archives/2014/02/mouse_event_bub.html
       if ('ontouchstart' in document.documentElement &&
          !$(parent).closest(Selector.NAVBAR_NAV).length) {
-        $('body').children().on('mouseover', '*', $.noop)
+        $('body').children().on('mouseover', null, $.noop)
```

Comment:
yes, but i thought i'd make it explicit (since that's what tripped up an earlier review)